packages/angular/projects/ui-angular/src/lib/primitives-poc/primitive/primitivie-component/primitive.component.ts

```diff
@@ -0,0 +1,24 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  ElementRef,
+  HostBinding,
+  Input,
+  OnInit,
+  Renderer2,
+  ViewEncapsulation,
+} from '@angular/core';
+
+@Component({
+  selector: 'primitive',
+  template: 'hi ',
+  encapsulation: ViewEncapsulation.None,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class AmplifyPrimitiveComponent {
```

Comment:
For my understanding, this is supposed to be the base interface for other primitives, right? If so, can this be better named?